pkg/systemlogmonitor/logwatchers/journald/log_watcher.go

```diff
@@ -135,27 +135,28 @@ func (j *journaldWatcher) watchLoop() {
 }
 
 const (
-	// defaultJournalLogPath is the default path of journal log.
-	defaultJournalLogPath = "/var/log/journal"
-
 	// configSourceKey is the key of source configuration in the plugin configuration.
 	configSourceKey = "source"
 )
 
 // getJournal returns a journal client.
 func getJournal(cfg types.WatcherConfig, startTime time.Time) (*sdjournal.Journal, error) {
 	// Get journal log path.
-	path := defaultJournalLogPath
-	if cfg.LogPath != "" {
-		path = cfg.LogPath
-	}
-	// If the path doesn't present, NewJournalFromDir will create it instead of
-	// returning error. So check the path existence ourselves.
-	if _, err := os.Stat(path); err != nil {
-		return nil, fmt.Errorf("failed to stat the log path %q: %v", path, err)
+	path := ""
+	var journal *sdjournal.Journal
+	var err error
+	if cfg.LogPath == "" {
+		journal, err = sdjournal.NewJournal()
+	} else {
+		// If the path doesn't exist, NewJournalFromDir will
+		// create it instead of returning error. So check the
+		// path existence ourselves.
+		if _, err = os.Stat(cfg.LogPath); err != nil {
+			return nil, fmt.Errorf("failed to stat the log path %q: %v", path, err)
+		}
+		// Get journal client from the log path.
+		journal, err = sdjournal.NewJournalFromDir(cfg.LogPath)
 	}
```

Comment:
Now we fall back to system default path, can you print a message here on what path it is used?

Reply:
+1 for the comment above. @stribb Do you have tiem to update this?

Reply:
I agree it would be nice. But there's no API to find that out as far as I can tell.

https://www.freedesktop.org/software/systemd/man/sd_journal_open.html#